docker/readpdf/Pipfile

```diff
@@ -6,6 +6,8 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
+pypdf2 = "*"
+pikepdf = "*"
 
 [requires]
 python_version = "3.9"
```

Comment:
lets make this 3.10 since the image is using 3.10

Reply:
Added that, thanks.